eth/ethconfig/config.go

```diff
@@ -166,13 +166,18 @@ type Config struct {
 	OverrideOptimismEcotone *uint64 `toml:",omitempty"`
 
 	OverrideOptimismInterop *uint64 `toml:",omitempty"`
+
+	// [Kroma: ZKT to MPT]
+	OverrideKromaMPT    *uint64 `toml:",omitempty"`
+	DisableMPTMigration bool
+
+	RollupHistoricalRPC        string
```

Comment:
I think not changing the order is easy to compare with OP when doing upstream works